### ISSUE TITLE
Pan - Enhancement Request

### ISSUE THREAD

**Issue by haasebert** [tbuser/thingiview.js#10]
Can you add panning capability?  It seems that it could be not too difficult, and there are many parts that are modeled off-center on Thingiverse.  Just a thought...

**Comment by tbuser** [comment]
Yeah I'm working on a new version of thingiview and it'll have panning.

**Comment by haasebert** [comment]
Cool. Thanks for replying!

-CH

**Comment by Pandryl** [comment]
Is the panning feature going to see light?

**Comment by TiagoRechau** [comment]
I really want it too, but i don't think it will
